docs/operations.md

```diff
@@ -297,6 +379,7 @@ To run the spy against mainnet:
 
 ```bash
 docker run \
+    --pull=always \
```

Comment:
Is this something that should get updated in docs site as well for starting the spy?

Reply:
@barnjamin yes absolutely. When @bruce-riley merged the PR for `--env` it didn't have that and the docs as they were broke.